src/html_files/index.ts

```diff
@@ -19,6 +19,7 @@ DataTypes.set('flamegraphs', {name: 'flamegraphs', hideClass: '', trueId: '', ca
 DataTypes.set('top_functions', {name: 'topfunctions', hideClass: '', trueId: '', callback: topFunctions});
 DataTypes.set('processes', {name: 'processes', hideClass: '', trueId: '', callback: processes});
 DataTypes.set('perfstat', {name: 'perfstat', hideClass: '', trueId: '', callback: perfStat});
+DataTypes.set('aperfstat', {name: 'aperfstat', hideClass: '', trueId: '', callback: aperfStat});
```

Comment:
All new data types need to be implemented with support for the hiding of all zeros/N/A graphs, which IIRC also means including support for pairing graphs in side-by-side comparisons.

Reply:
Are you sure about that? Your example report disagrees. All of the lines here that don't specify a hideClass and trueId don't support the things I'm saying should be required on all new data types going forward (when possible).

Reply:
Oh, the attached report has no graphs at all, which falls into a different category. Still, I'm ~fairly sure that the hideClass and trueId being empty means this does not support the features I am asking for

Reply:
Saw offline that this works, yay